config/application.rb

```diff
@@ -24,10 +24,7 @@ class Application < Rails::Application
 
     # our lib/ folder doesn't follow the Zeitwerk conventions for
     # autoloading, so rely the classic loader instead.
-    config.autoloader = :classic
-
-    config.autoload_paths += %W[#{config.root}/lib]
-    config.autoload_paths += Dir["#{config.root}/lib/**/"]
+    config.autoloader = :zeitwerk
```

Comment:
Would be good to say what's motivated this change in the commit message.

Reply:
done